src/components/Tab/types.ts

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
```

Comment:
This import doesn't look like it's needed

Reply:
Good catch.  I think this became unnecessary after changing some of the types in this file.